src/man/firejail-profile.txt

```diff
@@ -434,7 +447,9 @@ Make directory or file read-only.
 Make directory or file read-write.
 .TP
 \fBtmpfs directory
-Mount an empty tmpfs filesystem on top of directory. Directories outside user home or not owned by the user are not allowed. Sandboxes running as root are exempt from these restrictions.
+Mount an empty tmpfs filesystem on top of directory. Changes do not persist after termination.
+Directories outside user home or not owned by the user are not allowed. Sandboxes running as root are exempt from these restrictions.
+This directive has no effect for files (they appear unmodified and changes persist after termination).
```

Comment:
note: this one was surprising and might lead to unexpected behavior. but i'm just documenting the current state.